tests/fixture/tmpnet/network.go

```diff
@@ -325,26 +322,33 @@ func (n *Network) Create(rootDir string) error {
 	return n.Write()
 }
 
-// Starts all nodes in the network
-func (n *Network) Start(ctx context.Context, w io.Writer) error {
-	if _, err := fmt.Fprintf(w, "Starting network %s (UUID: %s)\n", n.Dir, n.UUID); err != nil {
-		return err
+// Starts the specified nodes
+func (n *Network) Start(ctx context.Context, w io.Writer, nodesToStart ...*Node) error {
+	nodesToWaitFor := nodesToStart
+	if len(nodesToStart) < len(n.Nodes) && nodesToStart[0] != n.Nodes[0] {
+		// Wait for all nodes to ensure health is logged for the bootstrap node
+		nodesToWaitFor = n.Nodes
+	} else {
+		// Simplify output by only logging network start for the first node or when all nodes are starting at once
```

Comment:
If all the nodes are starting at once, we're also starting the first node right?